ivy/functional/ivy/manipulation.py

```diff
@@ -549,7 +549,26 @@ def squeeze(
 
     >>> print(ivy.squeeze(x, axis=0).shape)
     (2, 2)
+    >>> print(ivy.squeeze(x).shape)
+    (2, 2)
 
+    >>> x = ivy.array([[[[1, 2, 3]], [[4, 5, 6]]]])
+    >>> print(x.shape)
+    (1, 2, 1, 3)
+    >>> print(ivy.squeeze(x, axis=2).shape)
+    (1, 2, 3)
+
+    >>> x = ivy.array([[[0], [1], [2]]])
+    >>> print(x.shape)
+    (1, 3, 1)
+    >>> print(ivy.squeeze(x))
```

Comment:
Sorry for being nitpicky, but can you please add a single newline after every print output in the docstrings? This is because the way `ivy_test/test_docstrings.py` works is that it assumes a print statement output ends at a newline.